### PR TITLE
Remove jQuery dependency for Post masonry and carousel.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Masonry block.
- Carousel block.

### Types of changes

- Improvement

### How has this been tested?

- Drag and drop masonry 
- Enable pagination and check infinite scroll and load more button pagination settings.
- Drag and drop carousel
- Enable equal height check editor as well as frontend. Equal hight working properly or not.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/js/post.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+let loadStatus = true;
+window.UAGBPostCarousel = {
+	_setHeight( scope ) {
 
-	window.UAGBPostCarousel = {
-		_setHeight( scope ) {
-			const post_wrapper = scope.find( '.slick-slide' ),
-				post_active = scope.find( '.slick-slide.slick-active' );
+		if( scope.length > 0 ){
+			const post_wrapper = scope[0].querySelectorAll( '.slick-slide' ),
```
Comment (RajkiranBagal) [inline assets/js/post.js:7]: Make all variables `camelCase`

File: blocks-config/post/class-uagb-post.php [hunk 2]
```diff
@@ -1254,17 +1254,47 @@ public function post_pagination() {
...
+		 * Required attribute for query.
+		 *
+		 * @param array $attributes plugin.
+		 * @return array of requred query attributes.
+		 * @since x.x.x
+		 */
+		public function required_attribute_for_query( $attributes ) {
+
+			return array(
+				'postsToShow'        => ( isset( $attributes['postsToShow'] ) ) ? esc_attr( $attributes['postsToShow'] ) : 6,
```
Comment (RajkiranBagal) [inline blocks-config/post/class-uagb-post.php:1285]: Escaping is done when we are Outputting something. For Inputs, we need to sanitize the data.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: assets/js/post.js (hunk 1 above)
Comment (AbhijitNage123) [reply assets/js/post.js]: Updated.

File: blocks-config/post/class-uagb-post.php (hunk 2 above)
Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php]: Updated.

File: assets/js/post.js [hunk 3]
```diff
@@ -1,195 +1,206 @@
...
+	_init( $attr, $selector ) {
+		let count = 2;
+		const windowHeight50 = window.innerHeight / 1.25;
+		let $scope = document.querySelector( $selector );
+		const loader = $scope.querySelectorAll( '.uagb-post-inf-loader' )
+		if ( 'scroll' === $attr.paginationEventType ) {
+			
+			window.addEventListener( 'scroll', function() {
+
+				const BoundingClientRect = $scope.querySelectorAll( '.uagb-post__items' )[0].lastElementChild.getBoundingClientRect();
```
Comment (RajkiranBagal) [inline assets/js/post.js:83]: camelCase these type of vars too

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: assets/js/post.js (hunk 3 above)
Comment (AbhijitNage123) [reply assets/js/post.js]: Updated.

**Review by supriyajjadhav — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: blocks-config/post/class-uagb-post.php [hunk 4]
```diff
@@ -1410,20 +1445,16 @@ public function add_post_dynamic_script() {
...
-							( function( $ ) {
-
-								var $scope = $( '.uagb-block-<?php echo esc_html( $key ); ?>' );
-								$scope.imagesLoaded( function() {
-									$scope.find( '.is-masonry' ).isotope();
-								});
-
-								$( window ).resize( function() {
-									$scope.find( '.is-masonry' ).isotope();
+							scope = document.querySelector( '.uagb-block-<?php echo esc_html( $key ); ?>' );
```
Comment (sandeshjangam) [inline blocks-config/post/class-uagb-post.php:1448]: scope is not defined here

Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php]: Updated.

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**